lib/spack/spack/report.py

```diff
@@ -245,6 +245,7 @@ def __init__(self, cls, function, format_name, args):
         self.cls = cls
         self.function = function
         self.filename = None
+        self.ctest_parsing = getattr(args, "ctest_parsing", False)
```

Comment:
@zackgalbreath @tldahlgren Where is `args.ctest_parsing` defined? I'm currently trying to refactor these classes to make them independent of `argparse` namespaces and for some reason I can't find references to this argument.